Add call out to promote Timescale Cloud (#1816)
timescale/docs: promscale/index.md

---

# Promscale
Promscale is a unified metric and trace observability backend for Prometheus,

> Jaeger and OpenTelemetry built on PostgreSQL and TimescaleDB.

Reviewer: ```suggestion
Jaeger, and OpenTelemetry. It is built on PostgreSQL and TimescaleDB.
```

> Promscale serves as a robust and 100% PromQL-compliant Prometheus remote

Reviewer: ```suggestion
Promscale provides robust and 100% PromQL-compliant Prometheus remote
```

> storage and as a durable and scalable Jaeger storage backend. Promscale is a
> certified Jaeger storage backend.

Reviewer: ```suggestion
storage. Promscale is a certified Jaeger storage backend that is durable and scalable.
```

Unlike other observability backends, it has a simple and easy-to-manage
architecture with just two components: the Promscale Connector and the

> Promscale Database (PostgreSQL with the TimescaleDB and Promscale extensions).

Reviewer: ```suggestion
Promscale database. The Promscale database consists of PostgreSQL with the 
TimescaleDB and Promscale extensions.
```

<highlight type="cloud" header="Promscale with Timescale Cloud" button="Get started for free"
to="https://console.cloud.timescale.com/signup?campaign=promscale&source=ps-docs-home">

> The best way to use Promscale is with Timescale Cloud, our fully hosted and managed

Reviewer: ```suggestion
The best way to use Promscale is with Timescale Cloud, a fully hosted and managed
```

database platform. Enjoy all the best features of Promscale without the

> hassle of managing your database—with automatic backups and failover, high

Reviewer: ```suggestion
hassle of managing your database. Timescale Cloud provides automatic 
backups and failover, high
```

availability, flexible scaling, security and data compliance, VPC peering, and

> much more. Get cost savings up to 94% compared to managed Prometheus offerings from

Reviewer: ```suggestion
much more. Get cost savings of up to 94% compared to managed Prometheus offerings from
```

> AWS, GCP and Grafana.

Reviewer: ```suggestion
AWS, Google Cloud Platform, and Grafana.
```

Author: I don't think we need to expand GCP in the same way we don't expand AWS.

Reviewer: In that case, we'll need to update our linter.

</highlight>

> See the [Promscale with Timescale Cloud installation guide][ptc-install] for details.

Reviewer: ```suggestion
For more information about using Promscale with Timescale Cloud, see the 
[Promscale installation guide][ptc-install] for details.
```
